project/scripts/winCmdTests

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
 set -e
-PREFIX="dist/target/pack"
+PREFIX="dist/win-x86_64/target/pack"
```

Comment:
Windows ? Shoudn't it be the linux plateform (this is a bash script) ?

Reply:
this is a windows test that is run on windows, it uses the bash that comes with Cygwin and the msys bash that comes with Git

Reply:
Indeed: https://github.com/scala/scala3/blob/04ada792f2cc81177c2f430e1a1b9060eab7b54c/.github/workflows/ci.yaml#L237